### ISSUE TITLE
chat.send() when no event is occurring

### ISSUE THREAD

**Issue by TheConfax** [coal0/python-omegle#3]
How can I send a message conditional to the fact that no event is occurring?
Eg: the partner is not talking, nothing is happening, so I send a specific message.

Sorry for the question but I'm a beginner ^^

**Comment by coal0** [comment]
This depends on your needs. What exactly do you mean when you say "nothing is happening"? Do you mean no:

1. Events of any type coming through;
2. Messages received;
3. 'Partner started typing' events;
4. Or something else entirely?

I assume you want this to trigger after x seconds of inactivity. Do you want it to happen concurrently - that is independently from your main loop, or as part of it?

**Comment by TheConfax** [comment]
I meant 1. (events of any type coming trough). I noticed that the code keeps waiting for an event so I cannot write and send messages in the same script.

I just wanted to have a fully functional chat and I have now solved it with threading, one loop handles events while another loop handles message sending.

I honestly don't know if that is the intended way to do stuff, I'm learning Python in free time, and I would love to have advice by some real coder, but I don't know if you are willing to review my code and I also suspect that what I'm doing could kinda be against Omegle TOS ^^
